release-notes/0.42.0.rst

```diff
@@ -0,0 +1,29 @@
+0.42.0 (2025-09-15)
+===================
+
+Upgrade Notes
+-------------
+
+- When :class:`.QiskitRuntimeService` is initialized without an instance 
+  and there is no saved account instance, there will now be a warning detailing 
+  the current filters (``tags``, ``region``, ``plans_preference``) and available instances. If ``plans_preference``
+  is not set, free and trial plan instances will be prioritized over paid instances. 
+
+  Additional warnings have also been added to make the current active insance more clear:
+
+      - Warning messages to tell users whether they're using a saved account or a new account.
+      - When retrieving a backend and an instance is automatically selected, there will be a warning with
+        the instance name and plan type. 
+
+  It is now also possible to initialize :class:`.QiskitRuntimeService` with just a token. Since the ``ibm_quantum``
```

Comment:
```suggestion
  It is now also possible to initialize :class:`.QiskitRuntimeService` with only a token. Since the ``ibm_quantum``
```
`only` is preferred over `just` for translatability